core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -571,7 +582,8 @@ final class ShuffleBlockFetcherIterator(
     }
   }
 
-  private def throwFetchFailedException(blockId: BlockId, address: BlockManagerId, e: Throwable) = {
+  private[storage] def throwFetchFailedException(
+      blockId: BlockId, address: BlockManagerId, e: Throwable) = {
```

Comment:
nit: if the method declaration is multi-line, each arg on its own line.  And this method should have had a return type in the first place (probably my fault, oops)

```scala
private[storage] def throwFetchFailedException(
    blockId: BlockId,
    address: BlockManagerId,
    e: Throwable): Unit = {
```